### PR TITLE
Beautify operator specifications

### PR DESCRIPTION
**Pull request by dkozykowski**
### Details:
 - update operator specifications to use Latex commands

### Tickets:
 - N/A

### CODE REVIEW

**Comment by lazarevevgeny** [comment]
@dkozykowski , could you update the PR description with the screenshots of rendered formulas after our documentation build to make sure that they are correctly rendered?

File: docs/ops/arithmetic/Maximum_1.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ As a first step input tensors *a* and *b* are broadcasted if their shapes differ
 After broadcasting *Maximum* does the following with the input tensors *a* and *b*:
 
 \f[
-o_{i} = max(a_{i}, b_{i})
+o_{i} = max(a_{i},\ b_{i})
```
Comment (dkozykowski) [inline docs/ops/arithmetic/Maximum_1.md:15]: old:

image

new:

image

File: docs/ops/arithmetic/Minimum_1.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 As a first step input tensors *a* and *b* are broadcasted if their shapes differ. Broadcasting is performed according to `auto_broadcast` attribute specification. As a second step *Minimum* operation is computed element-wise on the input tensors *a* and *b* according to the formula below:
 
 \f[
-o_{i} = min(a_{i}, b_{i})
+o_{i} = min(a_{i},\ b_{i})
```
Comment (dkozykowski) [inline docs/ops/arithmetic/Minimum_1.md:13]: old:

image

new:

image

File: docs/ops/arithmetic/Mod_1.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 As a first step input tensors *a* and *b* are broadcasted if their shapes differ. Broadcasting is performed according to `auto_broadcast` attribute specification. As a second step *Mod* operation is computed element-wise on the input tensors *a* and *b* according to the formula below:
 
 \f[
-o_{i} = a_{i} % b_{i}
+o_{i} = a_{i} \mod b_{i}
```
Comment (dkozykowski) [inline docs/ops/arithmetic/Mod_1.md:13]: old:

image

new:

image

Here old mod sign was not rendered at all. There's exactly the same problem with FloorMod, I'll update it in next commit.

File: docs/ops/comparison/GreaterEqual_1.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ Before performing arithmetic operation, input tensors *a* and *b* are broadcaste
 After broadcasting *GreaterEqual* does the following with the input tensors *a* and *b*:
 
 \f[
-o_{i} = a_{i} >= b_{i}
+o_{i} = a_{i} \geq b_{i}
```
Comment (dkozykowski) [inline docs/ops/comparison/GreaterEqual_1.md:40]: old:

image

new:

image

File: docs/ops/comparison/LessEqual_1.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ Before performing arithmetic operation, input tensors *a* and *b* are broadcaste
 After broadcasting *LessEqual* does the following with the input tensors *a* and *b*:
 
 \f[
-o_{i} = a_{i} <= b_{i}
+o_{i} = a_{i} \leq b_{i}
```
Comment (dkozykowski) [inline docs/ops/comparison/LessEqual_1.md:15]: old:

image

new:

image

File: docs/ops/comparison/NotEqual_1.md [hunk 6]
```diff
@@ -37,7 +37,7 @@ Before performing arithmetic operation, input tensors *a* and *b* are broadcaste
 After broadcasting *NotEqual* does the following with the input tensors *a* and *b*:
 
 \f[
-o_{i} = a_{i} != b_{i}
+o_{i} = a_{i} \neq b_{i}
```
Comment (dkozykowski) [inline docs/ops/comparison/NotEqual_1.md:40]: old:

image

new:

image

File: docs/ops/convolution/Convolution_1.md [hunk 7]
```diff
@@ -16,15 +16,15 @@ n_{out} = \left ( \frac{n_{in} + 2p - k}{s} \right ) + 1
...
   \f]
 *   Size of the receptive field of output feature:
   \f[
-  r_{out} = r_{in} + ( k - 1 ) * j_{in}
+  r_{out} = r_{in} + ( k - 1 ) \cdot j_{in}
   \f]
 *   Center position of the receptive field of the first output feature:
   \f[
-  start_{out} = start_{in} + ( \frac{k - 1}{2} - p ) * j_{in}
+  start_{out} = start_{in} + ( \frac{k - 1}{2} - p ) \cdot j_{in}
```
Comment (dkozykowski) [inline docs/ops/convolution/Convolution_1.md:27]: old:

image

new:

image

File: docs/ops/convolution/DeformableConvolution_1.md [hunk 8]
```diff
@@ -12,7 +12,7 @@ Output is calculated using the following formula:
 
   \f[
 
-  y(p) = \sum_{k = 1}^{K}w_{k}x(p + p_{k} + {\Delta}p_{k})
+  y(p) = \displaystyle{\sum_{k = 1}^{K}}w_{k}x(p + p_{k} + {\Delta}p_{k})
```
Comment (dkozykowski) [inline docs/ops/convolution/DeformableConvolution_1.md:15]: I'm not sure why, but currently in the old version, the formula is not rendered.

new:

image

File: docs/ops/convolution/DeformableConvolution_8.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ Output is calculated using the following formula:
 
   \f[
 
-  y(p) = \sum_{k = 1}^{K}w_{k}x(p + p_{k} + {\Delta}p_{k}) * {\Delta}m_{k}
+  y(p) = \displaystyle{\sum_{k = 1}^{K}}w_{k}x(p + p_{k} + {\Delta}p_{k}) \cdot {\Delta}m_{k}
```
Comment (dkozykowski) [inline docs/ops/convolution/DeformableConvolution_8.md:17]: Same as DeformableConvolution.

new:

image

File: docs/ops/logical/LogicalXor_1.md [hunk 10]
```diff
@@ -37,7 +37,7 @@ Before performing logical operation, input tensors *a* and *b* are broadcasted i
 After broadcasting *LogicalXor* does the following with the input tensors *a* and *b*:
 
 \f[
-o_{i} = a_{i} xor b_{i}
+o_{i} = a_{i} \oplus b_{i}
```
Comment (dkozykowski) [inline docs/ops/logical/LogicalXor_1.md:40]: old:

image

new:

image

File: docs/ops/logical/LogicalNot_1.md [hunk 11]
```diff
@@ -25,7 +25,7 @@
 *LogicalNot* does the following with the input tensor *a*:
 
 \f[
-a_{i} = not(a_{i})
+a_{i} = \lnot a_{i}
```
Comment (dkozykowski) [inline docs/ops/logical/LogicalNot_1.md:28]: old:

image

new:

image

File: docs/ops/pooling/AdaptiveAvgPool_8.md [hunk 12]
```diff
@@ -11,19 +11,19 @@ The kernel dimensions are calculated using the following formulae for the `NCDHW
...
+w_{start} &=& floor(k \cdot W_{in}/W_{out})\\
+w_{end}   &=& ceil((k+1) \cdot W_{in}/W_{out})
 \end{array}
 \f]
 
 The output is calculated with the following formula:
 
 \f[
-Output(i,j,k) = \frac{Input[d_{start}:d_{end}, h_{start}:h_{end}, w_{start}:w_{end}]}{(d_{end}-d_{start})*(h_{end}-h_{start})*(w_{end}-w_{start})}
+Output(i,j,k) = \frac{Input[d_{start}:d_{end}, h_{start}:h_{end}, w_{start}:w_{end}]}{(d_{end}-d_{start}) \cdot (h_{end}-h_{start}) \cdot (w_{end}-w_{start})}
```
Comment (dkozykowski) [inline docs/ops/pooling/AdaptiveAvgPool_8.md:26]: Same as DeformableConvolution.

new:

image

File: docs/ops/pooling/AdaptiveMaxPool_8.md [hunk 13]
```diff
@@ -11,12 +11,12 @@ The kernel dimensions are calculated using the following formulae for the `NCDHW
...
-h_{start} &=& floor(j*H_{in}/H_{out})\\
-h_{end}   &=& ceil((j+1)*H_{in}/H_{out})\\
-w_{start} &=& floor(k*W_{in}/W_{out})\\
-w_{end}   &=& ceil((k+1)*W_{in}/W_{out})
+d_{start} &=& floor(i \cdot D_{in}/D_{out})\\
+d_{end}   &=& ceil((i+1) \cdot D_{in}/D_{out})\\
+h_{start} &=& floor(j \cdot H_{in}/H_{out})\\
+h_{end}   &=& ceil((j+1) \cdot H_{in}/H_{out})\\
+w_{start} &=& floor(k \cdot W_{in}/W_{out})\\
+w_{end}   &=& ceil((k+1) \cdot W_{in}/W_{out})
```
Comment (dkozykowski) [inline docs/ops/pooling/AdaptiveMaxPool_8.md:19]: Same as DeformableConvolution.

new:

image

File: docs/ops/arithmetic/Floor_1.md [hunk 14]
```diff
@@ -10,7 +10,7 @@
 element in the output tensor with the following formula:
 
 \f[
-a_{i} = floor(a_{i})
+a_{i} = \lfloor a_{i} \rfloor
```
Comment (dkozykowski) [inline docs/ops/arithmetic/Floor_1.md:13]: old:

image

new:

image

File: docs/ops/arithmetic/Multiply_1.md [hunk 15]
```diff
@@ -11,7 +11,7 @@ Before performing arithmetic operation, input tensors *a* and *b* are broadcaste
 After broadcasting *Multiply* performs multiplication operation for the input tensors *a* and *b* using the formula below:
 
 \f[
-o_{i} = a_{i} * b_{i}
+o_{i} = a_{i} \cdot b_{i}
```
Comment (dkozykowski) [inline docs/ops/arithmetic/Multiply_1.md:14]: old:

image

new:

image

**Review by dkozykowski — COMMENTED** [review]
> @dkozykowski , could you update the PR description with the screenshots of rendered formulas after our documentation build to make sure that they are correctly rendered?

@lazarevevgeny, sure. I provided old version vs new version comparison.

File: docs/ops/pooling/AdaptiveAvgPool_8.md [hunk 16]
```diff
@@ -11,19 +11,19 @@ The kernel dimensions are calculated using the following formulae for the `NCDHW
...
+w_{start} &=& \lfloor k \cdot \frac{W_{in}}{W_{out}}\rfloor\\
+w_{end}   &=& \lceil(k+1) \cdot \frac{W_{in}}{W_{out}}\rceil
 \end{array}
 \f]
 
 The output is calculated with the following formula:
 
 \f[
-Output(i,j,k) = \frac{Input[d_{start}:d_{end}, h_{start}:h_{end}, w_{start}:w_{end}]}{(d_{end}-d_{start})*(h_{end}-h_{start})*(w_{end}-w_{start})}
+Output(i,j,k) = \frac{Input[d_{start}:d_{end}, h_{start}:h_{end}, w_{start}:w_{end}]}{(d_{end}-d_{start}) \cdot (h_{end}-h_{start}) \cdot (w_{end}-w_{start})}
```
Comment (dkozykowski) [inline docs/ops/pooling/AdaptiveAvgPool_8.md:26]: Same as DeformableConvolution.

new:


image

File: docs/ops/pooling/AdaptiveMaxPool_8.md [hunk 17]
```diff
@@ -11,12 +11,12 @@ The kernel dimensions are calculated using the following formulae for the `NCDHW
...
-h_{start} &=& floor(j*H_{in}/H_{out})\\
-h_{end}   &=& ceil((j+1)*H_{in}/H_{out})\\
-w_{start} &=& floor(k*W_{in}/W_{out})\\
-w_{end}   &=& ceil((k+1)*W_{in}/W_{out})
+d_{start} &=& \lfloor i \cdot \frac{D_{in}}{D_{out}}\rfloor\\
+d_{end}   &=& \lceil(i+1) \cdot \frac{D_{in}}{D_{out}}\rceil\\
+h_{start} &=& \lfloor j \cdot \frac{H_{in}}{H_{out}}\rfloor\\
+h_{end}   &=& \lceil(j+1) \cdot \frac{H_{in}}{H_{out}}\rceil\\
+w_{start} &=& \lfloor k \cdot \frac{W_{in}}{W_{out}}\rfloor\\
+w_{end}   &=& \lceil(k+1) \cdot \frac{W_{in}}{W_{out}}\rceil
```
Comment (dkozykowski) [inline docs/ops/pooling/AdaptiveMaxPool_8.md:19]: Same as DeformableConvolution.

new:

image

**Review by lazarevevgeny — APPROVED** [review]
Looks good. But we need review from @andrew-zaytsev as a primary doc owner.

**Review by ggalieroc — APPROVED** [review]

**Review by tsavina — APPROVED** [review]

**Pull request merged**